src/backend/loadPlan.js

```diff
@@ -38,6 +47,27 @@ export type LoadResult = {|
   +cred: TimelineCred,
 |};
 
+export function createPlan(
+  cache: CacheProvider,
+  githubToken: ?GithubToken,
+  pluginDeclarations: $ReadOnlyArray<PluginDeclaration>,
```

Comment:
Why does it take an array of plugin declarations, but hardcode the loaders? What happens if the user provides declarations that don't correspond to the hardcoded loader set?

Reply:
Noted for next attempt. Will use the same import-style dependency as the plugin loader itself uses.